src/pybind/mgr/volumes/fs/operations/volume.py

```diff
@@ -134,6 +134,9 @@ def delete_volume(mgr, volname, metadata_pool, data_pools):
         if r != 0:
             return r, outb, outs
     result_str = "metadata pool: {0} data pool: {1} removed".format(metadata_pool, str(data_pools))
+    result_str += "; if there are active snap-schedules associated with this "
```

Comment:
Does this still needs to warn users, since the timers and db connections are now cleaned up when a volume is deleted?

Reply:
Since we do have a small window between the instant the fs is deleted  by the MDS and the notification is handled by `snap-schedule` module that the timer might get triggered in the `snap-schedule` module, I think this warning still makes sense.

Reply:
Updated the warning message.
Also, added the warning message to `fs rm` command handler in the mon as well.